components/aws/sagemaker/tests/integration_tests/resources/definition/create_endpoint_pipeline.py

```diff
@@ -0,0 +1,55 @@
+import kfp
+from kfp import components
+from kfp import dsl
+from kfp.aws import use_aws_secret
+
+sagemaker_model_op = components.load_component_from_file("../../model/component.yaml")
+sagemaker_deploy_op = components.load_component_from_file("../../deploy/component.yaml")
+
+
+@dsl.pipeline(
+    name="Create Hosting Endpoint in SageMaker",
+    description="SageMaker deploy component test",
+)
+def create_endpoint_pipeline(
+    region="",
+    endpoint_url="",
+    image="",
+    model_name="",
+    endpoint_config_name="",
+    endpoint_name="",
+    model_artifact_url="",
+    variant_name_1="",
+    instance_type_1="",
+    initial_instance_count_1="",
+    initial_variant_weight_1="",
+    network_isolation="",
+    role="",
+):
+    create_model = sagemaker_model_op(
+        region=region,
+        endpoint_url=endpoint_url,
+        model_name=model_name,
+        image=image,
+        model_artifact_url=model_artifact_url,
+        network_isolation=network_isolation,
+        role=role,
+    ).apply(use_aws_secret("aws-secret", "AWS_ACCESS_KEY_ID", "AWS_SECRET_ACCESS_KEY"))
```

Comment:
We should no longer rely on hard-coded credentials.

Reply:
This will be cleaned up once @akartsky  adds the IAM role support feature.

Reply:
Yes, let's please change this in another PR.